VCFv4.3.tex

```diff
@@ -545,25 +545,39 @@ \subsection{VCF tag naming conventions}
 
 \section{INFO keys used for structural variants}
 \label{sv-info-keys}
-\begin{samepage}
 The following INFO keys are reserved for encoding structural variants.
 In general, when these keys are used by imprecise variants, the values should be best estimates.
-When a key reflects a property of a single alt allele (e.g.\ SVLEN), then when there are multiple alt alleles there will be multiple values for the key corresponding to each allele (e.g.\ SVLEN=-100,-110 for a deletion with two distinct alt alleles).
+When present, per allele keys must be specified for all alt alleles (including non-structural alleles).
+The missing value should used as a placeholder for the alt alleles in which the key does not have a meaningful value.
+The empty string is used to encode missing values in lists of strings.
+
+For compatibility with older VCF versions, 
 \footnotesize
 \begin{verbatim}
 ##INFO=<ID=IMPRECISE,Number=0,Type=Flag,Description="Imprecise structural variation">
+\end{verbatim}
+\normalsize
+
+Indicates that this record contains an imprecise structural variant $ALT$ allele.
+This field is a convenience field and can be inferred from the combination of the presence of $CIPOS$ and absence of $HOMSEQ$.
```

Comment:
I would clarify here if `IMPRECISE` is required in such cases or not.

Reply:
Flag fields are problematic since we can't have a flag per alt allele. The wording I've used essentially deprecates IMPRECISE, but keeps it around as a convience field for backward compatibility. I'd drop it entirely but it's used in existing pipelines.

Reply:
It looks to me that the most recent wording in this PR does _not_ deprecate `IMPRECISE`. We'll revisit `IMPRECISE` and its use in #508, and for now I'm going to mark this discussion as resolved